src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/index.tsx

```diff
@@ -56,19 +59,41 @@ const WorkspacePipelinePage: NextPageWithLayout = (props: Props) => {
     },
   });
 
-  useCacheKey(["pipelines", pipelineCode], () => refetch());
+  const clearCache = useCacheKey(["pipelines", pipelineCode], () => refetch());
+  // to prevent an "dirty read" from onSave function of isSchedulingEnabled state
+  const isEnabledRef = useRef<boolean>();
```

Comment:
I'm not sure to understand the difference between the isSchedulingEnabled and this ref.